docs/pages/upgrade-to-v3.md

```diff
@@ -16,11 +17,98 @@ The process to upgrade an app built with XMTP V2 to V3 is designed to be straigh
 
 - 🟢 **The agent upgrade path is ready**. For detailed guidance, [open an issue](https://github.com/xmtp/xmtp-js/issues) in the Node SDK GitHub repo.
 
+## Primary XMTP identifier is now a flexible identity object, not an Ethereum address
+
+XMTP is evolving from using Ethereum account addresses (0x...) as the primary identifier to a more flexible identity model. This change allows for broader support of different authentication mechanisms, including the currently supported Externally Owned Accounts (EOAs) and Smart Contract Wallets (SCWs), as well as future support for Passkeys.
+
+Instead of assuming an Ethereum address as the unique identifier, developers now define an identity object that explicitly includes the identity type (kind) and the identifier.
+
+With this new model, apps reference identities or inbox IDs, rather than Ethereum addresses. Some identity types (like Passkeys) do not have an associated address, so identities and inbox IDs provide a consistent way to identify users across different authentication methods.
```

Comment:
```suggestion
With this new model, apps reference users by inboxIDs, rather than Ethereum addresses. An inboxId will have a list of identities. Some identity types (like Passkeys) do not have an associated on chain address, so using the inbox IDs provide a consistent way to identify users across different authentication methods.
```

Reply:
[{ETHERUM, 0x23984937452}, {PASSKEY, askdfjaskdfhkajsd}]